easyapplybot.py

```diff
@@ -262,70 +281,174 @@ def click_button(self, xpath):
+                                        try:
+                                            log.info("Found test element %s", testLabel_locator)
+                                            text = testLabelElement.text
+                                            log.info("test element text: %s", text)
+                                            #assuming this question is asking if I am authorized to work in the US
+                                            if ("Are you" in text and "authorized" in text) or ("Have You" in text and "eduation" in text):
+                                                #Be sure to find the child element of the current test question section
+                                                yesRadio = testLabelElement.find_element(By.XPATH, yes_locator[1])
+                                                time.sleep(1)
+                                                log.info("Attempting to click the radio button for %s", yes_locator)
+                                                self.browser.execute_script("arguments[0].click()", yesRadio)
+                                                log.info("Clicked the radio button %s", yes_locator)
+
+                                            #assuming this question is asking if I require sponsorship
+                                            if "require" in text and "sponsorship" in text:
+                                                noRadio = testLabelElement.find_element(By.XPATH, no_locator[1])
+                                                time.sleep(1)
+                                                log.info("Attempting to click the radio button for %s", no_locator)
+                                                self.browser.execute_script("arguments[0].click()", noRadio)
+                                                log.info("Clicked the radio button %s", no_locator)
+
+                                            # assuming this question is asking if I require sponsorship
+                                            if "you have" in text and "Bachelor's" in text:
+                                                yesRadio = testLabelElement.find_element(By.XPATH, yes_locator[1])
+                                                time.sleep(1)
+                                                log.info("Attempting to click the radio button for %s", yes_locator)
+                                                self.browser.execute_script("arguments[0].click()", yesRadio)
+                                                log.info("Clicked the radio button %s", yes_locator)
+
+                                            #Some questions are asking how many years of experience you have in a specific skill
+                                            #Automatically put the number of years that I have worked.
+                                            if "How many years" in text and "experience" in text:
```

Comment:
I did the yes/no clicking option as well, and it works. However, I am having issues filling up the text box with years of experience. What about using Selenium4 options to find the text and find the textbox below? did you check if this work? maybe I am missing something. Let me know if you want to discuss it.

Reply:
So I tried for quite awhile to use selenium 4 but was unable to get it working. Very little documentation or examples for selenium4  relative locator potential issues.

I will double check this issue later today.
You should post the log output from the log file.

Reply:
@

> I did the yes/no clicking option as well, and it works. However, I am having issues filling up the text box with years of experience. What about using Selenium4 options to find the text and find the textbox below? did you check if this work? maybe I am missing something. Let me know if you want to discuss it.

Can you upload the log from this ?